transit/script/screen.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import tomllib
+from transit_function import read_dbf_and_groupby_sum, read_transit_assignments
+
+HWY_SCREENS = {
```

Comment:
should these go into a data file? or the config? seems not as appropriate to have them hardcoded in the script

Reply:
Should put them in toml file? or create new file for those data?